activityLogs.go

```diff
@@ -0,0 +1,90 @@
+package solus
+
+import "context"
+
+const (
+	ActivityLogEventAdditionalIPCreateRequested ActivityLogsEvent = "additional_ip_create_requested"
+	ActivityLogEventAdditionalIPCreateFailed    ActivityLogsEvent = "additional_ip_create_failed"
+	ActivityLogEventAdditionalIPCreated         ActivityLogsEvent = "additional_ip_created"
+
+	ActivityLogEventAdditionalIPDeleteRequested ActivityLogsEvent = "additional_ip_delete_requested"
+	ActivityLogEventAdditionalIPDeleteFailed    ActivityLogsEvent = "additional_ip_delete_failed"
+	ActivityLogEventAdditionalIPDeleted         ActivityLogsEvent = "additional_ip_deleted"
+
+	ActivityLogEventBackupsDeleted ActivityLogsEvent = "backups_deleted"
+
+	ActivityLogEventComputeResourceCreateRequested ActivityLogsEvent = "compute_resource_create_requested"
+	ActivityLogEventComputeResourceDeleteRequested ActivityLogsEvent = "compute_resource_delete_requested"
+
+	ActivityLogEventComputeResourceVMCreateRequested ActivityLogsEvent = "compute_resource_vm_create_requested"
+	ActivityLogEventComputeResourceVMCreateFailed    ActivityLogsEvent = "compute_resource_vm_create_failed"
+	ActivityLogEventComputeResourceVMCreated         ActivityLogsEvent = "compute_resource_vm_created"
+
+	ActivityLogEventComputeResourceVMDeleteRequested ActivityLogsEvent = "compute_resource_vm_delete_requested"
+
+	ActivityLogEventComputeResourceVMBatchDeleted ActivityLogsEvent = "compute_resource_vm_batch_deleted"
+
+	ActivityLogEventExternalIntegrationRequested ActivityLogsEvent = "external_integration_requested"
+
+	ActivityLogEventFailedToSendEmailNotification ActivityLogsEvent = "failed_to_send_email_notification"
+
+	ActivityLogEventIPBlockChanged ActivityLogsEvent = "ip_block_changed"
+
+	ActivityLogEventLocationCreated ActivityLogsEvent = "location_created"
+	ActivityLogEventLocationDeleted ActivityLogsEvent = "location_deleted"
+	ActivityLogEventLocationChanged ActivityLogsEvent = "location_changed"
+
+	ActivityLogEventUserCreateRequested ActivityLogsEvent = "user_create_requested"
+	ActivityLogEventUserCreateFailed    ActivityLogsEvent = "user_create_failed"
+	ActivityLogEventUserCreated         ActivityLogsEvent = "user_created"
+
+	ActivityLogEventUserDeleteRequested ActivityLogsEvent = "user_delete_requested"
+	ActivityLogEventUserDeleteFailed    ActivityLogsEvent = "user_delete_failed"
+	ActivityLogEventUserDeleted         ActivityLogsEvent = "user_deleted"
+)
+
+// ActivityLogsService provides access to Activity Logs.
+type ActivityLogsService service
+
+type ActivityLogsEvent string
```

Comment:
Please put this type definition above constants. It's a common way to define enums.